mlfinlab/portfolio_optimization/hcaa.py

```diff
@@ -174,14 +178,17 @@ def _get_optimal_number_of_clusters(self,
                 reference_correlation = np.array(reference_asset_returns.corr())
                 reference_distance_matrix = np.sqrt(2 * (1 - reference_correlation).round(5))
 
-                reference_cluster_assignments = cluster_func.fit_predict(reference_distance_matrix)
+                # reference_cluster_assignments = cluster_func.fit_predict(reference_distance_matrix)
```

Comment:
Why is this line commented out? Rather remove it if we aren't using it.

Reply:
Done